### PR TITLE
chore: add generate all command to makefile

### PR DESCRIPTION
**Pull request by kentakozuka**
For convenience...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,9 @@ docker-run:
 # Go
 #############################
 
+.PHONY: generate
+generate: proto-all mockgen
```
Comment (cre8ivejp) [inline Makefile:41]: The rule name is too abstract. It can be anything.
Please specify what it generates.

Comment (kentakozuka) [reply Makefile:41]: That's what I expect this command to run all generate commands.
Do you have any idea?

Comment (cre8ivejp) [reply Makefile:41]: If you expect to generate all the commands, how about `generate-all` or `generate-all-commands`?

Comment (kentakozuka) [reply Makefile:41]: `generate-all` sounds look to me 👍

**Comment by kentakozuka** [comment]
@cre8ivejp Fixed 👍

**Review by cre8ivejp — APPROVED** [review]
👍

**Pull request merged**